sections/desktop-img.liquid

```diff
@@ -0,0 +1,32 @@
+<div class="desktop-images">
+    <img src="{{section.settings.desktop_image_1 | img_url: '800x'}}">
+    <img src="{{section.settings.desktop_image_2 | img_url: '800x'}}">
+    <img src="{{section.settings.desktop_image_3 | img_url: '800x'}}">
+</div>
```

Comment:
xD ok my idea to build this with blocks is to iterate the blocks with liquid

[Rendering blocks](https://shopify.dev/tutorials/develop-theme-use-sections#rendering-section-blocks)